SharpDenizenTools/MetaObjects/MetaObject.cs

```diff
@@ -49,6 +49,16 @@ public abstract class MetaObject
         /// <summary>The backing <see cref="MetaDocs"/> instance.</summary>
         public MetaDocs Meta;
 
+        /// <summary>The original raw values specified for the object</summary>
+        public Dictionary<string, string> RawValues = new();
```

Comment:
RawValues cannot be accurately contained to a `Dictionary` like this, many keys (eg `Usages`, `Examples`, etc.) can validly have multiple entries under the same key

Reply:
Changed to a `List<string>` value